AttoBot/lambda_function.py

```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
```

Comment:
Yes, I can add the pending status indicator for the package tests. Also, I'm going to show the reverse dependency results separately as a comment as that could take an arbitrary amount of time to complete.